### DISCUSSION TITLE
[RELEASE] Shipment

### DISCUSSION THREAD

**Discussion by LexiAshe** [nzp-team/nzportable#710] [Map Releases]
I wanted to challenge myself to see if I could make an entire zombie map in one day. This was the result! It's Shipment from Call of Duty 4! I've tested it as much as I could but, I'm past my time limit now so there are bound to be lots of mistakes/bugs in this build, so please let me know if you spot any and I'll go back at a later date and refine the map into something a lot better :) Was mostly just an exercise for me, but thought it would be cool to release what I made too.

![Screenshot 2024-04-23 000138](https://github.com/nzp-team/nzportable/assets/139988968/6dee9605-b6b0-4ec2-a99a-b5fa62c03b1c)

[DOWNLOAD](https://github.com/nzp-team/nzportable/files/15069365/lexi_shipment.zip)

**Comment by headachez** [comment]
waypointing couldve used some work but otherwise its a really fun map :)
![Screenshot 2024-04-23 073631](https://github.com/nzp-team/nzportable/assets/114017494/f9d49923-fbb0-4188-a307-18c117b45703)
![Uploading Screenshot 2024-04-23 073349.png…]()

**Comment by MotoLegacy** [comment]
Heads up! This does not meet install standard as the download is not hosted on archive.org!

**Comment by M0THRAK** [comment]
Hey, I went ahead and fixed the waypoints on this map if anyone is interested in using them: [Updated Waypoints](https://archive.org/download/lexi-shipment-fixed-waypoints/Lexi_Shipment%20Fixed%20Waypoints.zip)
 
_This fixes zombie path-finding issues_

**Comment by nonMat** [comment]
Fun!!!
